activerecord/lib/active_record/model_schema.rb

```diff
@@ -181,6 +181,7 @@ module ModelSchema
       self.protected_environments = ["production"]
 
       self.ignored_columns = [].freeze
+      self.permitted_columns = [].freeze
```

Comment:
I still don't think `permitted` is a good prefix. If we absolutely need a prefix than `only_` could work.